### ISSUE TITLE
Dropdown with multi select: accessibility issues 

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#6075]
**Describe the bug**
There are still accessibility issues found on the multiselect dropdown.  related to https://github.com/infor-design/enterprise/issues/4847 and infor-design/enterprise-wc#132
Some issues that I have found but not limited to:
- role combobox is missing
- when using a screen reader, the focused selection does not say if its selected or not.
- aria-selected is missing on each options, but after selecting an option, the aria-selected is there.

**To Reproduce**

1. Go to https://main-enterprise.demo.design.infor.com/components/dropdown/test-configured-as-multiselect.html

**VPAT Recommendation**
Developers must ensure that combobox have appropriate role="Combobox" and necessary ARIA attributes.
See relevant infos:
http://whatsock.com/tsg/Coding%20Arena/ARIA%20Comboboxes/ARIA%20Comboboxes%20(Simulated,%20Readonly%20Multiselect)/demo.htm
https://www.w3.org/TR/wai-aria-practices/#combobox
https://dequelabs.github.io/combobo/demo/ (this has an example with an accessible dropdown with multiple selections)

**Version**

- ids-enterprise:  4.60

**Platform**
 - Infor Application/Team Name: Landmark Client
 - Device: (if applicable): NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: 85.0

**Additional context**
    https://jira.lawson.com/browse/LMCLIENT-34370

**Comment by tmcconechy** [comment]
cc @Jah25 - FYI for the new component

**Comment by janahintal** [comment]
QA Passed. Verified in https://main-enterprise.demo.design.infor.com/components/dropdown/test-configured-as-multiselect.html
![image](https://user-images.githubusercontent.com/45757016/159239758-cb3719b1-3d8d-4f90-8933-056ad2a5f3bf.png)
